api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1139,101 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Creating a subscription user subscribes to event-types and
```

Comment:
I feel the need to stress the key properties of a subscription. Wdyt of `Subscriptions allows applications to easily scale the number of clients by managing consumed event offsets and distributing load between instances.`